app/components/back-office-app-services/variables.tf

```diff
@@ -208,6 +208,11 @@ variable "tags" {
   type        = map(string)
 }
 
+variable "feature_key_vault_enabled" {

```

Comment:
Can we make this more descriptive? Something like "api_authorization_enabled" maybe? Bit confused about what it's for just reading the PR

Reply:
my only qualm with changing it to something like `api_authorization_enabled` is that even if that’s false, api auth checking is still technically happening, it’s just referencing a fixed test key that I’ve written here: https://github.com/Planning-Inspectorate/back-office/pull/2017/files#diff-43fca6de22bce10b10d2c788d167431d0ff290389e1953b2f76d0360a2c34905

Reply:
That's a good point - maybe something like api_key_vault_authorization_enabled? I think feature_key_vault_enabled is maybe a bit too vague, and it implies that we're using Key Vault or not - and we use Key Vault for a lot of different things.

Reply:
Yeah that makes sense. Cool - `api_key_vault_authorization_enabled` sounds good. Will switch it up to that!

Reply:
addressed in latest commit